src/Components/Blazor/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -20,8 +21,13 @@ public class RemoteAuthenticationService<TRemoteAuthenticationState, TProviderOp
          where TRemoteAuthenticationState : RemoteAuthenticationState
          where TProviderOptions : new()
     {
+        private const int _userCacheRefreshInterval = 60;
```

Comment:
Minor detail (since I know this is an implementation detail anyway), but could this be a `static TimeSpan` to avoid ambiguity about what units we're working in?

Reply:
Changed.